src/core/lib/channel/message_size_filter.c

```diff
@@ -38,13 +38,16 @@
 #include <grpc/support/log.h>
 #include <grpc/support/string_util.h>
 
+#include "src/core/ext/client_config/method_config.h"
 #include "src/core/lib/channel/channel_args.h"
 
 #define DEFAULT_MAX_SEND_MESSAGE_LENGTH -1  // Unlimited.
 // The protobuf library will (by default) start warning at 100 megs.
 #define DEFAULT_MAX_RECV_MESSAGE_LENGTH (4 * 1024 * 1024)
 
 typedef struct call_data {
+  int max_send_size;
```

Comment:
the value returned by `grpc_method_config_get_max_request_message_bytes`, which is ultimately assigned to this field is an `int32_t`. Should these two fields be of that type too?

Same for `max_recv_size` below.